app/Http/Middleware/ForceHttps.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Chompy\Http\Middleware;
+
+use Closure;
+use League\Uri\Schemes\Http;
+use function League\Uri\parse;
+
+class ForceHttps
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request $request
+     * @param  \Closure $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        // If running in a production environment, redirect all traffic to HTTPS
+        // and the correct canonical URL, e.g. never '*.herokuapp.com'!
+        if (config('app.env') == 'production') {
+            $canonicalHost = parse(config('app.url'))['host'];
+            $hasIncorrectHost = $request->header('Host') !== $canonicalHost;
+
+            if ($hasIncorrectHost || ! $request->secure()) {
```

Comment:
Aw I see now that you made this change from `$isInsecure` to `! $request->secure()` in Aurora too! I dig it!